june/configs/config_example_7.yaml

```diff
@@ -0,0 +1,39 @@
+title: Covid configuration example
```

Comment:
do we need that many config_examples? We should delete all ones that are not needed

Reply:
Otherwise all the other tests can use the 30 day one which is a bit nicer/more complete for the tests